src/app/components/DisplayCourse.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+import { Box, Button, Typography } from "@mui/material";
+import theme from "theme";
+import DisplayTas from "./tas";
+import { IdentifiableUsers } from "@interfaces/type";
+import { usePathname, useRouter } from "next/navigation";
+import { useOfficeHour } from "@hooks/oh/useOfficeHour";
+
+interface DisplayCourseProps {
+  tas: IdentifiableUsers;
+}
+const DisplayCourse = (props: DisplayCourseProps) => {
+  const { tas } = props;
+  const router = useRouter();
+  const pathname = usePathname();
+  const { course } = useOfficeHour();
+
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        gap: "32px",
+        marginTop: "24px",
+        height: "100vh",
+      }}
+    >
+      <Box>
+        <Typography
+          variant="h6"
+          color={theme.palette.text.primary}
+          sx={{ fontWeight: "bold" }}
+        >
+          Announcement
+        </Typography>
+        <Box
+          sx={{
+            padding: "16px 16px",
+            bgcolor: "#F8F9FF",
+            borderRadius: "16px",
+            marginY: "16px",
+          }}
+        >
+          <Typography variant="body2" color={theme.palette.text.secondary}>
+            {course.announcement}
+          </Typography>
+        </Box>
+      </Box>
+      <Box>
+        <Typography
+          variant="h6"
+          color={theme.palette.text.primary}
+          sx={{ fontWeight: "bold" }}
+        >
+          Location & Hours
+        </Typography>
+        <Box
+          sx={{
+            padding: "16px 16px",
+            gap: "10px",
+            flexDirection: "column",
+            display: "flex",
+            marginTop: "16px",
+          }}
+        >
+          <Typography variant="body2" color={theme.palette.text.primary}>
+            JCC 4th floor huddle room
+          </Typography>
+          <Typography variant="body2" color={theme.palette.text.primary}>
+            6 - 9 PM
+          </Typography>
+        </Box>
+      </Box>
+      <Box sx={{ display: "flex", flexDirection: "column", gap: "10px" }}>
+        <Box
+          sx={{
+            display: "flex",
+            flexDirection: "row",
+            justifyContent: "space-between",
+          }}
+        >
+          <Typography
+            variant="h6"
+            color={theme.palette.text.primary}
+            sx={{ fontWeight: "bold" }}
+          >
+            TAs
+          </Typography>
+          <Button
+            style={{
+              textTransform: "none",
+              padding: 0,
+            }}
+            onClick={() => {
+              router.push(`${pathname}/tas`);
+            }}
+          >
+            <Typography variant="subtitle2">View All</Typography>
+          </Button>
+        </Box>
+        <Box sx={{ padding: "16px, 16px", height: "48%", overflow: "auto" }}>
+          <DisplayTas tas={tas} />
+        </Box>
+        <Button
```

Comment:
If there are a lot of TAs then this button hovers above the navbar like this (for iPhone 12 Pro):
<img width="431" alt="Screenshot 2024-08-18 at 4 44 27 PM" src="https://github.com/user-attachments/assets/ea387a11-091b-4bb7-8a89-5d318d1ae7c2">

And for iPhone SE it looks like this on my end:
<img width="399" alt="Screenshot 2024-08-18 at 4 47 25 PM" src="https://github.com/user-attachments/assets/5ba7ea6b-ffe6-4f3f-a64a-7cdc01e0e192">